doc/source/coding-style/formatting-tools.rst

```diff
@@ -134,6 +135,7 @@ project:
 
 Pre-commit
```

Comment:
```suggestion
pre-commit
```
It's weird to have a heading start with a lower case, but this is the name used by the creators.